src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/utils/layout/index.tsx

```diff
@@ -115,3 +117,18 @@ export function setupGroupComponents(components: (ILayoutComponent | ILayoutGrou
   });
   return childComponents;
 }
+
+export function getLayouytsetForDataElement(instance: IInstance, datatype: string, layoutsets: ILayoutSets) {
+  let foundLayout: string;
+  const currentTaskId = instance.process.currentTask.elementId;
+  layoutsets.sets.forEach((layoutset: ILayoutSet) => {
```

Comment:
Could we use `find` instead of `forEach`? The syntax could be something like:
```suggestion
  const foundLayout = layoutsets.sets.find((layoutSet: ILayoutSet) => {
    if (layoutSet.dataType !== dataType) {
      return false
    }
    return layoutSet.task.find((taskId: string) => taskId === currentTaskId);
  }
```

Reply:
Done